testutils/src/main/java/com/microsoft/identity/internal/testutils/mocks/MockTokenCreator.java

```diff
@@ -64,6 +64,7 @@
     public static final String MOCK_ISSUER_SUFFIX_VALUE = "/v2.0";
     public static final Pattern CLOUD_DISCOVERY_ENDPOINT_REGEX = Pattern.compile("^https:\\/\\/login.microsoftonline.com\\/common\\/discovery\\/instance\\?api-version=1.1\\&authorization_endpoint=https%3A%2F%2Flogin.microsoftonline.com%2Fcommon%2Foauth2%2Fv2.0%2Fauthorize$");
     public static final Pattern MOCK_TOKEN_URL_REGEX = Pattern.compile("https:\\/\\/login.microsoftonline.com\\/.*");
+    public static final Pattern DEVICE_CODE_FLOW_AUTHORIZATION_REGEX = Pattern.compile("https:\\/\\/login.microsoftonline.com\\/common\\/oAuth2\\/v2.0\\/devicecode");
```

Comment:
Feels like I should see a reference to this constant somewhere...

Reply:
It's in SingleAccountOverloadsMockedTest.java, a testing class in msal for single account

Reply:
Is the tenant always `common`?